cli/cmd/setup.go

```diff
@@ -106,14 +44,67 @@ func appendValueToPrompt(promptString, value string) string {
 
 func init() {
 	rootCmd.AddCommand(setupCmd)
+}
+
+func runSetup(cmd *cobra.Command, args []string) {
+	var zabbixURL, zabbixUser, zabbixPassword, rundeckURL, rundeckAPIKey string
+	zabbixConfigKeyName := "zabbix"
```

Comment:
I'd prefer these inlined if their value never changes and write:

    zabbixURL = viper.GetString("zabbix.url")

Seems more readable cutting away the indirection.

Reply:
So it is either the value is defined in the file, where we pick it, or it is not, so we get values from the user on the cli. Also, the user should be able to change it during setup if they want.

Reply:
What do you think @osule ?

Reply:
I refer to the keys used to retrieve the values. I don't the user should change the keys themselves.

Reply:
I may have created a misunderstanding. The following snippet

    zabbixURL = viper.GetString("zabbix.url")

was suggested as a replacement for:
 
    zabbixURL = getConfigValueByKey(zabbixConfigKeyName, "url")

My comment was about the little value `getConfigValueByKey` adds in the code.

Reply:
okay, fixed that in new push.